app/build.gradle

```diff
@@ -119,56 +119,50 @@ dependencies {
     implementation project(':data')
 
     kapt(
-        "android.arch.lifecycle:compiler:$arch_component_version",
+        "androidx.lifecycle:lifecycle-compiler:$arch_component_version",
         "com.github.bumptech.glide:compiler:$glide_version",
         "com.google.dagger:dagger-compiler:$dagger_version",
         "com.google.dagger:dagger-android-processor:$dagger_version",
-        "com.github.hotchemi:permissionsdispatcher-processor:$permission_dispatcher_version"
+        "org.permissionsdispatcher:permissionsdispatcher-processor:$permission_dispatcher_version"
     )
 
     implementation(
         fileTree(dir: 'libs', include: ['*.jar']),
         "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version",
         "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version",
 
-        "com.android.support:support-v4:$android_support_version",
-        "com.android.support:appcompat-v7:$android_support_version",
-        "com.android.support:recyclerview-v7:$android_support_version",
-        "com.android.support:design:$android_support_version",
-        "com.android.support.constraint:constraint-layout:$constraint_layout_version",
+        "androidx.legacy:legacy-support-v4:$android_legacy_support_version",
+        "androidx.appcompat:appcompat:$android_support_version",
+        "androidx.recyclerview:recyclerview:$android_support_version",
+        "com.google.android.material:material:$android_material",
```

Comment:
@luongvo is this a replacement or newly added?

Reply:
@sleepylee replacement of `com.android.support:design` https://developer.android.com/jetpack/androidx/migrate
![image](https://user-images.githubusercontent.com/16315358/53354515-7c317780-3959-11e9-9e2c-53ce584b9b60.png)